src/components/board/PostingDetail.js

```diff
@@ -52,24 +58,34 @@ const PostingDetail = () => {
   ) : (
     <ThemeProvider theme={theme}>
       <Container>
-        <Title style={{ maxWidth: "700px", margin: "2rem auto" }}>
-          <div style={{ textAlign: "center" }}>
-            <h1>POSTING DETAIL</h1>
-            <h1>{title}</h1>
+        <Title>
+          <div>
+            <TitleText>{title}</TitleText>
           </div>
         </Title>
-        <Editor>
-          <div dangerouslySetInnerHTML={{ __html: content }} />
-        </Editor>
-        <Button
-          type="submit"
-          onClick={() => handleEditPosting(id, data.posting)}
-        >
-          Edit
-        </Button>
-        <Button type="submit" onClick={handleDeletePosting}>
-          Delete
-        </Button>
+        <Form>
+          <h1 style={{ color: "#A5B592" }}>POSTING DETAIL</h1>
+          <Editor>
+            <div dangerouslySetInnerHTML={{ __html: content }} />
+          </Editor>
+        </Form>
+        <Buttons>
+          <Button onClick={handleMoveBack}>Back</Button>
```

Comment:
```suggestion
          <Button onClick={() => navigate("/board")}>Back</Button>
```
handleEditPosting 함수와 handleMoveBack 함수가 한 줄 밖에 되지 않으니,
이런식으로 작성하는 것도 좋을 것 같은데 어떤 방식이 더 좋을까요-?